config/jobs/kubernetes-sigs/cluster-api/cluster-api-upgrades-main.yaml

```diff
@@ -0,0 +1,174 @@
+periodics:
+- name: periodic-cluster-api-e2e-workload-upgrade-1-17-1-18-main
+  labels:
+    preset-dind-enabled: "true"
+    preset-kind-volume-mounts: "true"
+  decorate: true
+  optional: true
+  always_run: false
+  branches:
+  # The script this job runs is not in all branches.
+  - ^master$
+  path_alias: sigs.k8s.io/cluster-api
```

Comment:
Those settings are for pre-submit jobs. 
You should remove them and instead add `interval` (IMO every 24h is more than fine for these jobs)
See e.g. https://github.com/kubernetes/test-infra/blob/28e6387bb414383260d3d876ae6f5c9c558e42e0/config/jobs/kubernetes-sigs/cluster-api/cluster-api-periodics-main.yaml#L26-L35

Reply:
Same for all the jobs